### PR TITLE
🐛 Use UTK Hyku build

### PR DESCRIPTION
**Pull request by jeremyf**
The primary change in Docker is moving from JDK 11 to 17 and improving the FITs xml config.

```
❯ git slog e9200061..f71b284f --no-merges
* e1c0510 — update fits xml Kirk Wang, (2022-12-01)
* 78eea74 — update jdk from 11 to 17 Kirk Wang, (2022-11-30)
* a404d11 — make cluster name a variable Rob Kaufman, (2022-11-19)
* d5745de — fix loki deploy Rob Kaufman, (2022-11-15)
```

Related to:

- https://github.com/scientist-softserv/palni-palci/issues/690

<details>
<summary>Commits since the reference point I'm bringing over from UTK</summary>

```
❯ git slog f71b284f.. --no-merges
* 99e6df9 — remove mimir, fix postgres Rob Kaufman, (2023-07-20) (origin/mimir_begon)
* 7e3d1c2 — add signoz to stack Rob Kaufman, (2023-06-21) (origin/signoz)
* abb6747 — build a new version of the LB with opentracing installed Rob Kaufman, (2023-06-21) (origin/loadbalancer_tracing)
* f91dddd — Update label_maker.rb to do description Rob Kaufman, (2023-06-22)
* 607051a — Update labels in label_maker.rb Crystal Richardson, (2023-06-22)
* e152146 — rotate mimir password Rob Kaufman, (2023-06-07)
* 5899db9 — try the zalando postgres opperator Rob Kaufman, (2023-06-07)
* d53fa35 — k8s notes. need to encrypt secrets to commit rest Rob Kaufman, (2023-05-18) (origin/smorgasbord_of_changes)
* 6a4d0e2 — aws loadbalancer, static ip support, nginx updates, mirmir, database backups and monitoring update. yes these should have been seperate Rob Kaufman, (2023-05-18)
| * e14543c — update syntax for deployment April Rieger, (2023-06-07)
| * 6fdca19 — Add issue template: dockerize application Diem ("Yeem") Tran, (2023-06-07)
| * fdd7967 — Add issue template: restore tests Diem ("Yeem") Tran, (2023-06-07)
| * 6596297 — Add issue template: project setup Diem ("Yeem") Tran, (2023-06-07)
| * b3fe36f — Updates to personnel from Sherwin April Rieger, (2022-12-17) (origin/glass-canvas-personnel-updates)
| * 66200ab — remove templates from legacy workflow Diem Tran, (2023-06-07)
| * e033ec4 — remove default issue template and add dev ops templates Diem Tran, (2023-06-02)
| * f58b80c — remove gitlab templates and add github templates Diem Tran, (2023-03-17)
| * bd50b88 — Updates the resource allocation based on the ticket April Rieger, (2023-06-05) (origin/695-update-resource-allocations)
|/
* b914ee8 — install ruby Rob Kaufman, (2023-04-21)
* c97f874 — install ruby Rob Kaufman, (2023-04-21)
* 23112df — working on workflows Rob Kaufman, (2023-04-21)
* 42d50a6 — working on label maker action Rob Kaufman, (2023-04-21)
* c806d99 — fix action Rob Kaufman, (2023-04-21)
* bcfc3c1 — label maker initial commit Rob Kaufman, (2023-04-21)
* f231359 — jemaloc base until hyrax gets it Rob Kaufman, (2023-03-24)
* 9c759e5 — deployment needs for r2-friends, move runners Rob Kaufman, (2022-11-28)
```

</details>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,11 +23,11 @@ COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 ARG HYKU_BULKRAX_ENABLED="true"
 RUN sh -l -c " \
   yarn install && \
-  RAILS_ENV=production SECRET_KEY_BASE=FAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKE DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile"
+  RAILS_ENV=production SECRET_KEY_BASE=FAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKE DB_ADAPTER=nulldb DB_URL='postgresql://fake' bundle exec rake assets:precompile"
 RUN ln -sf /app/samvera/branding /app/samvera/hyrax-webapp/public/branding
 
-CMD ./bin/web
```
Comment (bkiahstroud) [inline Dockerfile:19]: If this `CMD` is removed, how does the rails server start in the `web` container?

Comment (jeremyf) [reply Dockerfile:19]: Well…how did UTK do it?  (I'm restoring this)

Comment (bkiahstroud) [reply Dockerfile:19]: It's possible it comes from this line? 

```
FROM ghcr.io/scientist-softserv/dev-ops/samvera:f71b284f as hyku-base
```

It's possible there's a `CMD` line in that file that gets inherited. 

If the server started without it I think it's fine, I was mostly asking out of curiosity (my other comment was the true "requested change")

**Review by bkiahstroud — APPROVED** [review]
I spun this up locally and successfully created a work with two PDFs :shipit:

**Pull request merged**